routes/admin/profile.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var router = express.Router();
+var session = require('express-session');
+
+var adminsModel = require('../../schemas/admins');
+var portalsModel = require('../../schemas/portals');
+var settingsModel = require('../../schemas/settings');
+var coursesModel = require('../../schemas/courses');
+
+
+router.use(function (req, res, next) {
```

Comment:
`renderState` is already declared in `index.js`. So, these lines have to be removed.